src/ExNavigationStack.js

```diff
@@ -435,6 +435,16 @@ class ExNavigationStack extends PureComponent<any, Props, State> {
     );
   }
 
+  _getNavigationBarHeight(latestRouteConfig) {
+    let height = NavigationBar.DEFAULT_HEIGHT;
+
+    if (latestRouteConfig.statusBar && latestRouteConfig.statusBar.translucent) {
+      height = NavigationBar.DEFAULT_HEIGHT_WITHOUT_STATUS_BAR + 24;
```

Comment:
@satya164 updated the PR. I can't reproduce the 1px off thing in my app anymore. 25 works in my app for now.